content/tutorials/getting-started/index.md

```diff
@@ -15,21 +15,27 @@ lastmod: 2021-08-30T16:00:00+02:00
 
 In this tutorial, we'll describe every step to help you run your first test with Gatling Enterprise.
 
-{{< alert tip >}}
+{{< alert info >}}
 **Requirements**
 
 * A valid [GitHub](https://github.com/) account
 * A Java Development Kit (JDK) installed in version 8 or 11
-* The [Gatling bundle](https://gatling.io/open-source/#downloadgatling) as a starter or an existing Maven/Gradle/SBT Gatling project
+{{< /alert >}}
+
+{{< alert tip >}}
+**Recommendations**
+
+* Using the [Gatling bundle](https://gatling.io/open-source/#downloadgatling) is beginner friendly
+* For long term projects, consider using a Gradle, Maven or SBT Gatling project, you can download some Gatling Enterprise ready projects from the Cloud dashboard directly!
```

Comment:
* 2 sentences: `project, you` => `project. You`
* `projects` => `sample projects`
* Click on the `Sample simulations` button in the top right corner.